server/middleware/security/authentication.js

```diff
@@ -84,9 +84,24 @@ export const checkJwtExpiry = (req, res, next) => {
 export const _authenticateUserByJwt = (req, res, next) => {
   if (!req.jwtPayload) return next();
   const userid = Number(req.jwtPayload.sub);
+  const lastLogin = req.jwtPayload.last;
```

Comment:
You set `lastLogin` here but are still using `req.jwtPayload.last` later. Maybe you should try to always use `lastLogin` once it's set?

Reply:
Agreed! I thought for some reason that perhaps we wouldn't want to be obvious about the naming of the last login timestamp in a JWT, since they can be decoded, but that's probably just me overthinking 😅